airflow/models/baseoperator.py

```diff
@@ -173,8 +173,6 @@ class derived from this one results in the creation of a task object,
         DAGS. Options can be set as string or using the constants defined in
         the static class ``airflow.utils.WeightRule``
     :type weight_rule: str
-    :param queue: specifies which task queue to use
```

Comment:
Good spot! But could you please remove the other one? This one matches the position of the queue param

Reply:
@ashb I moved the one I left to the proper position. It was more descriptive to me.